go/signedexchange/signedexchange.go

```diff
@@ -14,8 +14,22 @@ import (
 	"github.com/WICG/webpackage/go/signedexchange/cbor"
 	"github.com/WICG/webpackage/go/signedexchange/internal/bigendian"
 	"github.com/WICG/webpackage/go/signedexchange/mice"
+	"github.com/WICG/webpackage/go/signedexchange/version"
 )
 
+const HeaderMagicBytesLen = 8
+
+func HeaderMagicBytes(v version.Version) []byte {
+	switch v {
+	case version.Version1b1:
+		return []byte("sxg1-b1\x00")
+	case version.Version1b2:
+		return []byte("sxg1\x00\x00\x00\x00")
```

Comment:
This should be `"sxg1-b2\x00"`

Reply:
I thought the spec said the magic header byte is `"sxg1\x00\x00\x00\x00"`.

> 8 bytes consisting of the ASCII characters “sxg1” followed by 4 0x00 bytes, to serve as a file signature. This is redundant with the MIME type, and recipients that receive both MUST check that they match and stop parsing if they don’t.

Reply:
Done, but I'd like to know if the spec is correct or not.